src/traces/sankey/plot.js

```diff
@@ -125,13 +125,14 @@ module.exports = function plot(gd, calcData) {
 
     var linkHoverFollow = function(element, d) {
 
+        var rootBBox = gd.getBoundingClientRect();
```

Comment:
this matches the pattern we've had to use elsewhere. It's possible that *both* `getBoundingClientRect` calls could be avoided by figuring out the position directly from the link/node element (or by caching it in the element's d3 data or something) but if you use `getBoundingClientRect` for one you'd better use it for both. That could be an optimization later, this is good for now.